### PR TITLE
Malware fixes - vol 2

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N.a.

## Description
Indicator Extraction + adding required outputs + fixing validation issues.

## Screenshots
N.a.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml [hunk 1]
```diff
@@ -636,15 +636,49 @@ tasks:
...
+      description: commands.local.cmd.extract.indicators
+      script: Builtin|||extractIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - '17'
+    scriptarguments:
+      text:
```
Comment (David-BMS) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml:656]: Change to inline

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_SIEM_ingestion_-_Get_Incident_Data.yml [hunk 2]
```diff
@@ -780,15 +776,49 @@ tasks:
...
+      script: Builtin|||extractIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - '20'
+    scriptarguments:
+      text:
+        simple: ${CrowdStrike.Detection}
```
Comment (David-BMS) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_SIEM_ingestion_-_Get_Incident_Data.yml:797]: Change to inline

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_5.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### CrowdStrike Falcon - SIEM ingestion Get Incident Data
+- Playbook improvements.
```
Comment (David-BMS) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_5.md:4]: Please add a more detailed description

**Review by David-BMS — APPROVED** [review]

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml (hunk 1 above)
Comment (ssokolovich) [reply Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml:656]: Done

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_SIEM_ingestion_-_Get_Incident_Data.yml (hunk 2 above)
Comment (ssokolovich) [reply Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_SIEM_ingestion_-_Get_Incident_Data.yml:797]: Done

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_5.md (hunk 3 above)
Comment (ssokolovich) [reply Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_5.md:4]: Will add more context once the build is ok

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml file. @ssokolovich - please confirm the following is ok:
- line 398, 482, 637: the description for Set Endpoint information to layout task is "commands.local.cmd.set.incident"
- line 702: the description for Extract Indicators task is "commands.local.cmd.extract.indicators"

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml file. @ssokolovich - please confirm the following:
- line 283: updated task name to "Map behavior data to context", left description as-is
- line 371, 456, 492, 564: Set Endpoint information to layout task description is "commands.local.cmd.set.incident" (line 492 task name is Set Process Details To Layout)
- line 621: removed extra comma from simple value
- line 647: Extract Indicators task description is "commands.local.cmd.extract.indicators"

**Comment by julieschwartz18** [comment]
Doc review complete for this file. @ssokolovich - please confirm the following:
- line 251: the incidentfield should be Agent ID, not Agents ID
- line 314, 460, 547: the Set Severity task description is "commands.local.cmd.set.incident", also for Set Endpoint information to layout
- line 788: description for Extract Indicators task is "commands.local.cmd.extract.indicators"

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_-_Host_Advanced_Hunting_For_Network_Activity.yml file. @ssokolovich - please confirm: the following tasks all have the description commands.local.cmd.set.incident:
- line 790:  Update Tag - 'Malicious IP' 
- line 1069:  Update tag - 'Malicious DNS' 
- line 1165:  Update tag - "Remote Connections"
- line 1201:  Update Tag - ' Local FW Rules Change '
- line 1357: Update tag - "DNS Requests" 
- line 1440:  Update Tag - ' Enumeration of Network Shares '
- line 1476:  Update Tag - ' hosts File Modification'
- line 1512:   Update Tag - 'File External Connections'

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_Malware_-_Investigation_and_Response.yml file. @ssokolovich - please confirm:
- line 1268: the description for Set Tag `Suspicious Command-line` is "commands.local.cmd.set.incident"

**Pull request merged**